packages/markers/src/browser/problem/problem-widget.tsx

```diff
@@ -61,6 +61,14 @@ export class ProblemWidget extends TreeWidget {
         return;
     }
 
+    protected handleClickEvent(node: TreeNode | undefined, event: React.MouseEvent<HTMLElement>): void {
+        if (MarkerNode.is(node)) {
+            this.model.revealNode(node);
+        } else {
+            super.handleClickEvent(node, event);
```

Comment:
Should not it be called always to handle selection properly?

Reply:
Thanks, I updated it and re-tested it works well still :)